common/web/gesture-recognizer/src/tools/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# Builds the include script for the current Keyman version.
+#
+
+# Exit on command failure and when using unset variables:
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$(dirname $THIS_SCRIPT)"
+
+display_usage ( ) {
+  echo "Usage: $0 [configure] [clean] [build] [test]"
+  echo "          [--verbose|-v]"
+  echo "       $0 -h|--help"
+  echo
+  echo "  clean                  removes build/ folder"
+  echo "  build                  builds wrapped version of package"
+  echo "                           [if required will: configure]"
+}
+
+build ( ) {
+  echo "Building recorder page"
+  ./recorder/build.sh build
+  echo "Building unit-test resources module"
+  ./unit-test-resources/build.sh build
+  echo "Extracting fixture"
+  ./host-fixture/extract-fixture.sh > ../../build/tools/host-fixture.html
+  cp ./host-fixture/gestureHost.css ../../build/tools/gestureHost.css
+}
```

Comment:
With the changes in #6986 you should split this into separate `:target`s I think. Later, we'll add dependency checks.